openmdao/core/tests/test_coloring.py

```diff
@@ -157,7 +270,7 @@ def test_simul_coloring(self):
 
     def test_simul_coloring_example(self):
 
-        from openmdao.api import Problem, IndepVarComp, ExecComp, pyOptSparseDriver
+        from openmdao.api import Problem, IndepVarComp, ExecComp, ScipyOptimizeDriver
         import numpy as np
 
         SIZE = 10
```

Comment:
should the "must be an even number" comment appear here, in the user documentation version?

Reply:
fixed